src/pages/Search/Search.js

```diff
@@ -140,6 +163,53 @@ export default class Search extends Component {
           });
       }
   }
+  handleDateOptionDisplay = (selectedDateValue) => {
+    console.log("YOLO");
+    console.log('selectDateObject ' + selectedDateValue);
+    console.log(typeof selectedDateValue);
+    const {from, to} = this.state;
+    const modifiers = {start: from, end: to};
+    if (selectedDateValue === undefined) return;
+    console.log('dateOption Choice ' + selectedDateValue);
+    //trigger={<button> Trigger</button>}
+    if (selectedDateValue === 'dayOfWeek') {
+        return (
+            <>
+              <div className="headingRow"> Day of the Week </div>
+              <Popup open = {selectedDateValue === 'dayOfWeek'} position="right center">
+                  <Select className="dropdown"
+                    value={this.state.daysOfWeek}
+                    onChange={this.handleChange8}
+                    options={daysOfWeekOptions}
+                  />
+              </Popup>
+
+            </>
+        );
+      } else if (selectedDateValue === 'individualDates') {
+        return (
+              <Popup open = {selectedDateValue === 'individualDates'} position="right center">
```

Comment:
I'm a bit confused here. Is there a reason why `individualDates` pops up but `dateRange` doesn't?